src/main/java/kitchenpos/application/dto/response/MenuResponse.java

```diff
@@ -4,7 +4,7 @@
 import java.util.List;
 import java.util.stream.Collectors;
 import kitchenpos.domain.menu.Menu;
-import kitchenpos.domain.menu.MenuProduct;
+import kitchenpos.domain.menu.menu_product.MenuProduct;
```

Comment:
자바 패키지는 
`Package names are written in all lower case to avoid conflict with the names of classes or interfaces.`
이런 이유로 모두 소문자로 작성한다고 해요.
관련 [레퍼런스](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html) 남깁니다.